sdk/cosmos/azure-cosmos-spark_3-1_2-12/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 4.16.0-beta.1 (Unreleased)
 
 #### Features Added
+* Added an option to use raw json when applying partial updates via WriteStrategy `ItemPatch`- See [PR 32610](https://github.com/Azure/azure-sdk-for-java/pull/32610)
```

Comment:
Do we also need to add the documentation on how the rawJson will be identified (like what you mentioned in the PR description) ?
Or is it all implementation details that are not worth mentioning?

Reply:
Right now we only have Annie's PDF document (which I can't really change) - but, yes - we also should add documentation like Theo added for throughput control. Will bring it up in next Java sync-up - at least the PR is something we can point folks to. And this change is to unblock a customer scenario